.gitignore

```diff
@@ -3,3 +3,19 @@ docassemble/assemblylinewizard/docassemble*
 .history/
 .vscode/
 *.egg-info
+
+# alkiln testing
+tests/chromedriver
+*DS_Store*
+node_modules*
+.env
+npm-debug.log
+error*.jpg
+screenshot_*.jpg
+downloads_*
```

Comment:
Maybe give these a path? those are the only ones that seem at risk of a namespace collision someday

Reply:
Do you think we'll really have files with these names at the root of the project? If so I'll remove those and just manage it manually. It would take some changes to the library to change these filenames.

Reply:
Just to clarify - since the names don't have wildcards at the front, I think they'll only match to files at the root of the project. I'll try to do some more research on it, but if that's the case are they fine?

Reply:
Your current rule doesn't only match the root.

Reply:
Wasn't familiar with this and good to learn!
> If there is a [/] separator at the beginning or middle (or both) of the pattern, then the pattern is relative to the directory level of the particular .gitignore file itself. Otherwise the pattern may also match at any level below the .gitignore level.
-- https://git-scm.com/docs/gitignore#_pattern_format

Will adding a `/` at the beginning be a sufficient precaution?

Reply:
Yep. Would have added as a suggestion but I wasn't sure what path they belonged in.